src/EditorFeatures/TestUtilities/RenameTracking/MockPreviewDialogService.cs

```diff
@@ -30,7 +32,30 @@ public MockPreviewDialogService()
         {
         }
 
-        public Solution PreviewChanges(string title, string helpString, string description, string topLevelName, Glyph topLevelGlyph, Solution newSolution, Solution oldSolution, bool showCheckBoxes = true)
+        public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
+            => this;
+
+        public Task<Solution> PreviewChangesAsync(
+            string title,
+            string helpString,
+            string description,
+            string? topLevelName,
+            Glyph topLevelGlyph,
+            Solution newSolution,
+            Solution oldSolution,
+            CancellationToken cancellationToken)
+            => PreviewChangesAsync(title, helpString, description, topLevelName, topLevelGlyph, newSolution, oldSolution, showCheckBoxes: true, cancellationToken);
```

Comment:
How many existing callers were there that don't need checkboxes? Easier to just update all of them?

Reply:
It's only used for PreviewChangesSuggestedAction, which is used by the lightbulb code actions for preview changes. I don't think we want to change that experience just to simplify here? 

Other callers could all pass in true explicitly I guess. I'm not opposed to updating there. There's only 6